app/flatpak-cli-transaction.c

```diff
@@ -332,11 +332,22 @@ progress_changed_cb (FlatpakTransactionProgress *progress,
           g_autofree char *text = NULL;
           int row;
 
-          formatted_max = g_format_size (max);
-          if (transferred < 1024) // avoid "bytes"
-            formatted = g_strdup ("1.0 kB");
+          if (max < 1000) // avoid "bytes"
+            {
+              formatted = g_format_size (1000);
```

Comment:
Could get rid of the if blocks and do `formatted = transferred < 1000 ? g_format_size (1000) : g_format_size (transferred);` and similarly for the max.

Reply:
I find it better too.

Reply:
I find it better too.